app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_senter: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
```

Comment:
Use the same code style for identation. Choose one between 2 ways(start from a new line every argument, or this style) and use it everywhere